src/cfml/system/modules_app/system-commands/commands/browse.cfc

```diff
@@ -12,9 +12,9 @@ component {
 	/**
 	 * @URI.hint The URI to open as you would type it into your browser's address bar
```

Comment:
Where is the hint for the browser param?  If you add a param, you must also add the hint!  Also please add something to the help text in this command to show using the browser parameter.